SubModule.cs

```diff
@@ -54,6 +54,9 @@ private void AddModels(CampaignGameStarter gameStarter)
                         Settings.Instance.PregnancyDurationTweakEnabled || Settings.Instance.FemaleOffspringProbabilityTweakEnabled ||
                         Settings.Instance.TwinsProbabilityTweakEnabled)
                     gameStarter.AddModel(new TweakedPregnancyModel());
+                if (Settings.Instance.DefaultAgeTweakEnabled)
+                    gameStarter.AddModel(new TweakedDefaultAgeModel());
```

Comment:
For consistency that should be TweakedAgeModel.cs
Since we provide our own Model which inherits from the appropriate DefaultModel.

Reply:
yeah but it is actually easier if we use the full model name incase they add a new one. Obviously this is a bad example. I think it is a mistake not to include the full model name.

Reply:
It's a fair point, but it doesn't really matter what the base model is.
Also if you look at the code you will see that Taleworlds always has an abstract class AgeModel and then they provide an implementation called DefaultAgeModel.

Since at the end of the day our TweakedModel needs to implement that `Abstract Class` either by inheriting from the base DefaultModel (taleworlds implementation) or by directly inheriting from the abstract class.

It's unnecessary to mention the base class as part of the TweakedModel name.
Since we already mention the abstract class AgeModel: TweakedAgeModel.

Reply:
Also this way we could for example then have YoungTweakedAgeModel
OldTweakedAgeModel etc :)

Reply:
Shouldn't it be TweakedOldAgeModel or TweakedYoungAgeModel ? :)

Reply:
I would be okay with both ways for the Old/Young example, but it really depends on what we are trying to signify.

If for that example we would treat the different models as presets for the AgeModel.
Then I think OldTweakedAgeModel where Old refers to the preset and YoungTweakedAgeModel would be a bit better/more clear then TweakedYoungAgeModel/TweakedOldAgeModel?

Naming is the bane of Software Engineering 🗡️ 

Yes please proceed with the changes, will definitely help with the consistency 👍 


Maybe this argument will help why I feel it's unnecessary :)

Whether we inherit from the Base or implement the Abstract Model directly, is an implementation Detail that doesn't need to be exposed as part of the class name. Since we are free to do either.

If you think about it we are not tweaking the base class we are Tweaking the AgeModel whether we rely on using some default values (DefaultModel) or not, makes no difference.
Since our TweakedModel will be the Union of our choosen values and any potentially not Tweaked values. So our TweakedModel will represent a TweakedAgeModel.

Now if Taleworlds ever changes the Abstract Class name then we certainly should change our class name to reflect that.